types/summernote/index.d.ts

```diff
@@ -0,0 +1,240 @@
+			air?: popoverAirDef;
+		}
+
+		interface ModuleOptions {
+			options?: any; // todo
+		}
+
+		interface CreateLinkOptions {
+			text: string;
+			url: string;
+			newWindow: boolean;
+		}
+
+		type EditImageCallback = ($image: JQuery.Node) => void;
+	}
+
+	interface JQuery {
+		summernote(): JQuery;
+		summernote(command: string): JQuery;
+		summernote(options?: Summernote.Options): JQuery;
+		summernote(command: string, markupString: string): JQuery;
+		summernote(command: string, value: number): JQuery;
+		summernote(command: string, node: JQuery.Node): JQuery;
+		summernote(command: string, url: string, filename?: (string | Summernote.EditImageCallback)): JQuery;
+
+		summernote(command: 'destroy'): JQuery;
+		summernote(command: 'code', markupStr?: string): JQuery;
+		summernote(command: 'editor.pasteHTML' | 'pasteHTML', markup: string): JQuery;
+
+		// Basic API
+		summernote(command: 'editor.createRange' | 'createRange'): JQuery;
+		summernote(command: 'editor.saveRange' | 'saveRange'): JQuery;
+		summernote(command: 'editor.restoreRange' | 'restoreRange'): JQuery;
+		summernote(command: 'editor.undo' | 'undo'): JQuery;
+		summernote(command: 'editor.redo' | 'redo'): JQuery;
+		summernote(command: 'editor.focus' | 'focus'): JQuery;
+		summernote(command: 'editor.isEmpty' | 'isEmpty'): boolean;
+		summernote(command: 'reset'): JQuery;
+		summernote(command: 'disable'): JQuery;
+		summernote(command: 'enable'): JQuery;
+		// Font style API
+		summernote(fontStyle: 'editor.bold' | 'bold'): JQuery;
+		summernote(fontStyle: 'editor.italic' | 'italic'): JQuery;
+		summernote(fontStyle: 'editor.underline' | 'underline'): JQuery;
+		summernote(fontStyle: 'editor.strikethrough' | 'strikethrough'): JQuery;
+		summernote(command: 'editor.superscript' | 'superscript'): JQuery;
+		summernote(command: 'editor.subscript' | 'subscript'): JQuery;
+		summernote(command: 'editor.removeFormat' | 'removeFormat'): JQuery;
+		summernote(command: 'backColor', color: string): JQuery;
+		summernote(command: 'foreColor', color: string): JQuery;
+		summernote(command: 'fontName', fontName: string): JQuery;
+		summernote(command: 'editor.fontSize' | 'fontSize', fontSize: number): JQuery;
+		// Paragraph API
+		summernote(command: 'editor.justifyLeft' | 'justifyLeft'): JQuery;
+		summernote(command: 'editor.justifyRight' | 'justifyRight'): JQuery;
+		summernote(command: 'editor.justifyCenter' | 'justifyCenter'): JQuery;
+		summernote(command: 'editor.justifyFull' | 'justifyFull'): JQuery;
+		summernote(command: 'insertParagraph'): JQuery;
+		summernote(command: 'editor.insertOrderedList' | 'insertOrderedList'): JQuery;
+		summernote(command: 'editor.insertUnorderedList' | 'insertUnorderedList'): JQuery;
+		summernote(command: 'editor.indent' | 'indent'): JQuery;
+		summernote(command: 'editor.outdent' | 'outdent'): JQuery;
+		summernote(command: 'formatPara'): JQuery;
+		summernote(command: 'formatH1'): JQuery;
+		summernote(command: 'formatH2'): JQuery;
```

Comment:
Since the rule says you merge some function declarations, for example, you can replace the two with:
```ts
summernote(command: 'formatH1' | 'formatH2'): JQuery
```